File: C++/MathematicalPrograms/2DShapes/AreaAndPerimeterOfRectangle.cpp
//Program to find Area and Perimeter of Rectangle
#include <iostream>
#include <string>

using namespace std;

int main() {
	int Length, Breadth;
	double area, perimeter;
	cout << "Enter a Length measure";
	cin >> Length;
	cout << "Enter a Breadth measure";
	cin >> Breadth;
	area = Length*Breadth;
	perimeter = 2*(Length+Breadth);
	cout << "Perimter = " << perimeter << " cm";
	cout << "\nArea = " << area << " sq.cm.";
}

File: Java/MathematicalPrograms/2DShapes/AreaAndPerimeterOfRectangle.java
//Program to find Area and Perimeter of Rectangle
import java.util.Scanner;
public class AreaAndPerimeterOfRectangle {
	public static void main(String[] args) {
		Scanner myObj = new Scanner(System.in);
		System.out.println("Enter a Length measure");
		int length = myObj.nextInt();
		System.out.println("Enter a Breadth measure");
		int breadth = myObj.nextInt();
		int area = length*breadth;
		int perimeter = 2*(length+breadth);
		System.out.println("Perimter = " + perimeter + " cm");
		System.out.println("Area = " + area + " sq.cm.");
	}
}